### ISSUE TITLE
It's not possible to read csv without TextLineReader?

### ISSUE THREAD

**Issue by kitovyj** [tensorflow/tensorflow#5887]
`def input_data(start_index, amount, shuffle):
    
    data_folder = '/media/sf_vb-shared/data/'
    range_queue = tf.train.range_input_producer(amount, shuffle = shuffle)
    range_value = range_queue.dequeue()

    abs_index = tf.add(range_value, tf.constant(start_index))    
    abs_index_str = tf.as_string(abs_index, width = 9, fill = '0')
    
    png_file_name = tf.string_join([tf.constant(data_folder), tf.constant('data'), abs_index_str, tf.constant('.png')])
    csv_file_name = tf.string_join([tf.constant(data_folder), tf.constant('data'), abs_index_str, tf.constant('.csv')])

    csv_data = tf.read_file(csv_file_name)
    csv_data = tf.Print(csv_data, [csv_data], message = "This is csv_data: ")
    label_defaults = [[0.0] for x in range(n_classes)]   
    unpacked_labels = tf.decode_csv(csv_data, record_defaults = label_defaults)
    labels = tf.pack(unpacked_labels)
    labels = tf.Print(labels, [labels], message = "These are labels: ")`

The output is "This is csv_data: [0,0,0,1,0\n]", so
csv_data is read ok, but "These are labels" never gets printed out...
Is it possible to feed decode_csv with read_file output?

**Comment by prb12** [comment]
https://www.tensorflow.org/versions/r0.11/resources/index.html
> For help and support, technical or algorithmic questions, please submit your questions to Stack Overflow: https://stackoverflow.com/questions/tagged/tensorflow. You may also find answers in our FAQ, our glossary, or in the shapes, sizes and types guide. Please do not use the mailing list or issue tracker for support.
